.eslintrc.js

```diff
@@ -19,76 +19,38 @@ module.exports = {
     browser: true, // Enable browser global variables
     node: true, // Enable node global variables & Node.js scoping
     es2020: true, // Add all ECMAScript 2020 globals and automatically set the ecmaVersion parser option to ES2020
-    mocha: true, // Add Mocha testing global variables
   },
 
   plugins: [
     '@typescript-eslint', // Add some TypeScript specific rules, and disable rules covered by the typechecker
     'import', // Add rules that help validate proper imports
-    'mocha', // Add rules for writing better Mocha tests
-    'prettier', // Allows running prettier as an ESLint rule, and reporting differences as individual linting issues
   ],
 
-  extends: [
-    // ESLint recommended rules
-    'eslint:recommended',
+  extends: ['@xpring-eng/eslint-config-base/loose'],
 
-    // Add TypeScript-specific rules, and disable rules covered by typechecker
-    'plugin:@typescript-eslint/eslint-recommended',
-    'plugin:@typescript-eslint/recommended',
+  rules: {},
 
-    // Add rules for import/export syntax
-    'plugin:import/errors',
-    'plugin:import/warnings',
-    'plugin:import/typescript',
-
-    // Add rules for Mocha-specific syntax
-    'plugin:mocha/recommended',
-
-    // Add Airbnb + TypeScript support
-    'airbnb-base',
-    'airbnb-typescript/base',
+  overrides: [
+    {
+      files: 'test/**/*.test.ts',
+      rules: {
+        // We use non-null assertions liberally in tests to allow TypeScript to build
+        '@typescript-eslint/no-non-null-assertion': 'off',
 
-    // Add rules that specifically require type information using our tsconfig
-    'plugin:@typescript-eslint/recommended-requiring-type-checking',
+        // Because of gRPC, our tests have to have a ton of imports
+        'import/max-dependencies': ['warn', { max: 11 }],
 
-    // Enable Prettier for ESLint --fix, and disable rules that conflict with Prettier
-    'prettier/@typescript-eslint',
-    'plugin:prettier/recommended',
-  ],
+        // Because of gRPC, our tests have to have a ton of statements to be useful
+        'max-statements': ['warn', { max: 40 }],
 
-  rules: {
-    // This rule is about explicitly using `return undefined` when a function returns any non-undefined object.
-    // However, since we're using TypeScript, it will yell at us if a function is not allowed to return `undefined` in its signature, so we don't need this rule.
-    'consistent-return': 'off',
+        // TODO:(@hbergren) We should be able to get rid of these two rules eventually,
```

Comment:
nit: can you be more specific about what needs to happen before we can get rid of the rules? When can we expect this TODO to be revisited?